### PR TITLE
Added section on benefits of ZKP proofs

### PR DESCRIPTION
**Pull request by ken-ebert**
This addresses part 3 of Issue #8

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -662,7 +662,45 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 the cost of implementation complexity.
       </dd>
     </dl>
-
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats can only reveal all
+attribute values in a verifiable credential.
```
Comment (TallTed) [inline index.html]: It took some work to extract the meaning of this sentence --
> Many proof formats can only reveal all attribute values in a verifiable credential.

Perhaps something like this?
> For instance, many proof formats cannot reveal selected attribute values in a verifiable credential; they can only reveal all (or none).

File: index.html [hunk 2]
```diff
@@ -662,7 +662,45 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
+associated values that are required to satisfy the presentation requirements. 
+    </p>
+    <p>
+ZKP predicate proofs are a type of proof associated with an attribute.
+Predicate proofs include comparisons, such as
+"greater than", "less than", "not equal", "range", 
+"set membership", and "set non-membership".
+Predicate proofs can be
+constructed without requiring the issuer to create special fields, such as
+"age-under-18", age-over-21", "age-over-60", in the verifiable credential
```
Comment (TallTed) [inline index.html]: Missing double-quote before `age-over-21",`

File: index.html (hunk 1 above)
Comment (ken-ebert) [reply index.html]: I like your suggestion. I will update the text.

File: index.html (hunk 2 above)
Comment (ken-ebert) [reply index.html]: Yes. I will update the text.

File: index.html (hunk 1 above)
Comment (ken-ebert) [reply index.html]: Added @TallTed suggestion.

File: index.html (hunk 2 above)
Comment (ken-ebert) [reply index.html]: Added @TallTed suggestion.

File: index.html [hunk 3]
```diff
@@ -758,7 +758,46 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
+verifiable presentations, such as selective disclosure and predicate proofs.
+    </p>
+    <p>
+Selective disclosure is the ability of a holder to reveal a subset of
+the attributes of a verifiable credential in a verifiable presentation.
+A presentation based on ZKPs only contains those attributes and
+associated values that are required to satisfy the presentation requirements. 
+    </p>
+    <p>
+ZKP predicate proofs are a type of proof associated with an attribute.
```
Comment (brentzundel) [inline index.html]: "ZKP predicate proofs" -> Zero-knowledge predicate proofs

File: index.html [hunk 4]
```diff
@@ -758,7 +758,46 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
+Predicate proofs include comparisons, such as
+"greater than", "less than", "not equal", "range", 
+"set membership", and "set non-membership".
+Predicate proofs can be
+constructed without requiring the issuer to create special fields, such as
+"age-under-18", "age-over-21", "age-over-60", in the verifiable credential
+at the time the credential was issued.
+The holder can create a predicate proof based on "birth-date" in the
```
Comment (brentzundel) [inline index.html]: Recommended change:
For example, if a holder has a credential with a "birth-date" claim, 
the holder can create a predicate proof based on "birth-date" in the
verifiable presentation. A predicate value derived from "birth-date" in
the verifiable credential can cryptographically prove to the verifier that 
the holder's age is greater (or less) than a specified number, without 
revealing the holder's "birth-date".

File: index.html [hunk 5]
```diff
@@ -758,7 +758,46 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
...
+verifiable presentation derived from the verifiable credential to
+cryptographically prove to the verifier that the holder's age is greater than
+(or less than) a specified number without revealing the holders "birth-date".
+    </p>
+    <p>
+Verifiable credentials based on ZKPs are also quantum-resistant after
+presentation.
+    </p>
+    <p>
+A drawback of ZKPs is that they are more complex and larger than some
```
Comment (brentzundel) [inline index.html]: I am not a fan of `ZKPs` and generally prefer a phrase such as `zero-knowledge proof mechanisms` which better ensures that the language surrounding the phrase remains accurate.

**Review by brentzundel — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (ken-ebert) [reply index.html]: Added suggested text

File: index.html (hunk 4 above)
Comment (ken-ebert) [reply index.html]: Added suggested text

File: index.html (hunk 5 above)
Comment (ken-ebert) [reply index.html]: Made changes to reflect this suggestion.

**Review by brentzundel — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Pull request merged**